### DISCUSSION TITLE
Release 0.4.5 (1.20.40.22)

### DISCUSSION THREAD

**Discussion by minecraftchloe** [Mojang/minecraft-editor#143] [Announcements]
_[Read the previous release notes here](https://github.com/Mojang/minecraft-editor/discussions/140)_ 

--- 

## Summary 

We’re back after a few weeks of focusing on our first iteration of Edit/Test to share with you! Read on for more details 😀!  

As a reminder of Editor API support:  

- Editor Extensions [overview article](https://learn.microsoft.com/en-us/minecraft/creator/documents/editorextensionsintroduction) and  [API reference documentation](https://aka.ms/EditorAPI) on learn.microsoft.com 
- [Starter Kit](https://github.com/Mojang/minecraft-editor-extension-starter-kit): Contains the compiler tools and various libraries required to get started making Editor Extensions 
- [Samples](https://github.com/Mojang/minecraft-editor-extension-samples) created by our team to jump-start your own Extension creations 

--- 

## Introducing this release 

We heard your feedback that the current testing workflow is bulky, time-consuming, and takes away valuable time you could be building instead of restarting Minecraft to test minimal changes. Introducing the first iteration towards our v0.5: Edit/Test Lite release!  

**Here’s the need-to-know:**  
- Full round-trip edit --> (play)test experience within the context of the Editor  
- Customize your world conditions for testing 
- Changes you make in the test session will not be saved 
- Marketplace and persona are disabled in the test experience, which is also standard within the Editor 
- Available in single-player (for now 😊) - our goal is to make this available on dedicated server and in multiplayer sessions.  

There will be many iterations on this feature – beyond even v1.0. From our initial testing, we found that this cut down testing sequences from ~5 minutes to ~1 minute. Let us know what you think!  

---

## Fixed this week 

- Fixed a bug in the Selection tool where the Y axis of the Vector3 origin exceeded the dimension height range 
- Fixed a bug where blocks couldn’t be placed in the last block placed location after an undo 

 

--- 

## Next  

We follow semantic versioning and will be in v0.4.X for a bit. Moving forward there will be 2 concurrent workflows the team will be engaged with:  

1. Iterating on the Editor API: Source from our internal punch-lists and playtests, requirements as we add more features to the core Editor experience, and of course...your feedback! 

2. Evolving existing _and_ adding new features to the core editing experience.  

The next update is v0.5: Edit/Test Lite. This week is our first iteration towards that! We’ll have some smaller features within the context of v0.5, but the major focus is the Edit/Test flow.  

--- 

## Feedback  

Please keep sending any and all feedback you have. Now that we are transitioning out of summer vacations and into the fall/winter season, development is going to start picking up on our side!  

Additionally, if you're new to the Editor GitHub this week, we encourage you to join the Bedrock Add-Ons Discord [#Editor channel](https://discord.gg/TZnQKFwGpt)!  

Thank you for being here! You're the best! ❤️

**Comment by AbbottMc** [comment]
Nice Update!!!! Can't wait to see the 0.5.0 release!

**Comment by AndroAGENT** [comment]
How to open the new Minecraft Editor is it minecraft?:editor or what

**Reply by minecraftchloe** [reply]
Great question! Directions here: https://github.com/Mojang/minecraft-editor#accessing-the-editor
